resources/queries/targetedms/SampleFileAssayResults.query.xml

```diff
@@ -0,0 +1,39 @@
+<query xmlns="http://labkey.org/data/xml/query">
+    <metadata>
+        <tables xmlns="http://labkey.org/data/xml">
+            <table tableName="SampleFileAssayResults" tableDbType="NOT_IN_DB">
+                <columns>
+                    <column columnName="Id">
+                        <isHidden>true</isHidden>
+                    </column>
+                    <column columnName="FileName">
+                        <url>/targetedms-showPrecursorList.view?id=${Id}</url>
+                    </column>
+                    <column columnName="PeptideGroupCount">
```

Comment:
PeptideGroupCount and MoleculeListCount are really two different names for the same underlying thing in the DB. We choose one based on the type of data that's in the files. Here, where we can't easily conditionalize, I think we're better off having a single column with title "Proteins/Molecule Lists"